guides/doc-Quickstart/master.adoc

```diff
@@ -22,7 +22,7 @@ include::common/modules/ref_supported-operating-systems.adoc[]
 The installation will require 4GB of memory.
 For more information, see {InstallingServerDocURL}system-requirements_{project-context}[System Requirements].
 
-The Foreman installer uses Puppet (5 or later required) to install Foreman. This guide assumes that you have a newly installed operating system, on which the installer will setup Foreman, a Puppet server, and the Smart Proxy by default. It’s not advisable to follow the steps below on an existing system, since the installer will affect the configuration of several components.
+The Foreman installer uses Puppet to install Foreman. This guide assumes that you have a newly installed operating system, on which the installer will setup Foreman, a Puppet server, and the Smart Proxy by default. It’s not advisable to follow the steps below on an existing system, since the installer will affect the configuration of several components.
```

Comment:
Can we follow one sentence per line here?

Reply:
Applied in a separate commit.